.github/workflows/FissionBiome.yml

```diff
@@ -33,15 +33,15 @@ jobs:
           cd fission
           bun install
 
-      - name: Linter
-        id: linter-validation
+      - name: Lint
+        id: lint-validation
         if: ${{ always() }}
         run: |
           cd fission
           bun run lint && echo "Biome Linting Validation Passed" || (echo "Biome Linting Validation Failed" && exit 1)
 
-      - name: Prettier
-        id: prettier-validation
+      - name: Format
+        id: format-validation
```

Comment:
IDs not necessary (and kind of confusing imo) unless they're referenced later

Reply:
I'll remove them